README.md

```diff
@@ -123,6 +129,17 @@ instance to use the screen sharing functionality you will have to
 provide HTTPS access to both the files and the Janus gateway, like shown
 in the [deployment instructions](DEPLOYMENT.md).
 
+## Troubleshooting
+
+When jangouts server do not work, there is wide variety of ways how to ways how to debug it.
```

Comment:
I see your point, but it could confuse people about Jangouts having some kind of backend (apart from Janus). Anyway, it's fine with me as it's better than before :)